extras/dump_iphone_video.py

```diff
@@ -6,7 +6,7 @@
 import datetime
 import logging
 from typing import Optional
-from neurobooth_os.log_manager import make_default_logger
+from neurobooth_os.log_manager import make_db_logger, APP_LOG_NAME
```

Comment:
One thing I learned when doing PR #321, when doing `from Y.Z import X`, the value of `X` will not change when the module-level variable changes. (`X` becomes a separate variable---whatever it happens to be at import time.) 

Instead, if you do `import Y.Z as Z` and then later reference `Z.X`, you will see changes to the global module state.

Reply:
That's good to know, but in this case APP_LOG_NAME is a "constant"